src/tests/clipperUI/components/modeButtonSelector_tests.tsx

```diff
@@ -161,14 +161,14 @@ export class ModeButtonSelectorTests extends TestModule {
 			strictEqual(buttonElements[3].id, TestConstants.Ids.bookmarkButton, "The fourth button should be the bookmark button");
 		});
 
-		test("The tabbing should flow in element order, assuming they are all available", () => {
+		test("The aria-posinset attribute should flow in element order, assuming they are all available", () => {
 			let startingState = MockProps.getMockClipperState();
```

Comment:
There should be other tests in this file that describe various groups where the buttons are different. Tests for those scenarios will need to be added as well. Right now your code will only work in the base-case where the user is clipping a normal web page, but what about pdf/etc?